index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- Primary Meta Tags -->
+    <title>Linkbrary</title>
+    <meta name="title" content="Linkbrary" />
+    <meta name="description" content="세상의 모든 정보를 쉽게 저장하고 관리해보세요" />
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://linkbrary.com/" />
+    <meta property="og:title" content="Linkbrary" />
+    <meta property="og:description" content="세상의 모든 정보를 쉽게 저장하고 관리해보세요" />
+    <meta property="og:image" content="./preview-logo.svg" />
+
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content="https://linkbrary.com/" />
+    <meta property="twitter:title" content="Linkbrary" />
+    <meta property="twitter:description" content="세상의 모든 정보를 쉽게 저장하고 관리해보세요" />
+    <meta property="twitter:image" content="preview-logo.svg" />
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/reset.css" />
+    <link rel="stylesheet" href="./src/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" class="logo" />
+        </a>
+        <a class="cta cta-short" href="signin.html">
+          <span>로그인</span>
+        </a>
+      </nav>
+      <div class="hero-header">
+        <h1 class="slogan">
+          <span class="slogan-gradient background-clip-text"> 세상의 모든 정보</span>를
+          <br class="pc-br"/>
+          쉽게 저장하고
+          <br class="tablet-br"/> 
+          관리해 보세요
```

Comment:
옵셔널하게 br을 넣어주는 방향도 좋지만 flex를 통해 자연스럽게 wrap 되도록 하는 방법도 있으니 참고만 해주세요.
```suggestion
         <p style="display: flex; flex-wrap: wrap; justify-content: center;">
            <span style="display: inline-block;">쉽게 저장하고</span>
            <span style="display: inline-block;">관리해 보세요</span>
          </p>
```